keybinding-generator/generate-keybindings.mts

```diff
@@ -116,15 +116,15 @@ function compileKeybinding(opts: { key: string; command?: string; when?: string;
       key: replaceAll(key, "meta", "alt"),
       command,
       when: addWhenCond(when, "config.emacs-mcx.useMetaPrefixAlt"),
-      ...(args ? { args } : {}),
```

Comment:
This technique is not necessary.
When `args` is `undefined`, the key is omitted from the serialized JSON anyway.